contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -449,6 +449,9 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     // Pay out the DON fee to all transmitters
     // Bounded by "maxNumOracles" on OCR2Abstract.sol
     address[] memory transmitters = _getTransmitters();
+    if (transmitters.length == 0) {
+      return;
```

Comment:
Someone would need to trigger disperse before even setting any OCR config, correct? Seems like a reasonable check but maybe better to revert with an error?

Reply:
@bolekk We encountered this issue because `_disperseFeePool` is called when the OCR config is first set because it is called by the `_beforeSetConfig` hook.  Austin and I debugged this during a sync this morning.

Reply:
What's the reason for calling it in _beforeSetConfig?

Reply:
I'll defer to @justinkaseman to provide the best answer here.

Reply:
The context is:
NOP payments are held together in a lumped pool of funds to avoid the having user paying for N storage writes of gas for their request to fulfill.  
If the OCR config changes we need to pay out the this pool in case a NOP is being added or removed.